cobalt/cobalt.cc

```diff
@@ -45,41 +54,109 @@ content::ContentMainRunner* GetContentMainRunner() {
 static base::AtExitManager* g_exit_manager = nullptr;
 static cobalt::CobaltMainDelegate* g_content_main_delegate = nullptr;
 static PlatformEventSourceStarboard* g_platform_event_source = nullptr;
+
+// Toggle-only Switches.
+const auto kCobaltToggleSwitches = std::to_array<const char*>({
+  // Disable first run experience, kiosk, etc.
+  "disable-fre",
+  switches::kNoFirstRun,
+  switches::kKioskMode,
+  // Enable Blink to work in overlay video mode
+  switches::kForceVideoOverlays,
+  // Disable multiprocess mode.
+  switches::kSingleProcess,
+  // Accelerated GL is blanket disabled for Linux. Ignore the GPU blocklist
+  // to enable it.
+  switches::kIgnoreGpuBlocklist,
+  // This flag is added specifically for m114 and should be removed after
+  // rebasing to m120+
+#if BUILDFLAG(IS_ANDROID)
+  switches::kUserLevelMemoryPressureSignalParams,
+#endif  // BUILDFLAG(IS_ANDROID)
+  sandbox::policy::switches::kNoSandbox
+});
+
+// Switches with parameters. May require special handling with passed in values.
+const auto kCobaltParamSwitches = base::CommandLine::SwitchMap({
```

Comment:
I doubt this is trivially destructible go/cstyle#Static_and_Global_Variables

Reply:
fixed